packages/core/src/sdk/performance/useTTI.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react'
+
+// Time To Interactive - https://web.dev/articles/tti
+export default function useTTI() {
+  const [isInteractive, setIsInteractive] = useState(false)
+
+  useEffect(() => {
+    if ('PerformanceObserver' in window) {
+      let lastTaskEnd = 0
+      const TTI_TIMEOUT = 5000 // 5 seconds without long tasks as a criterion for TTI
```

Comment:
I think we can move this constant outside the component to avoid recreating it every time the `useEffect` runs.

Reply:
done [here](https://github.com/vtex/faststore/pull/2604/commits/2618f4da30bf148005d42c3f05185e1a6d8a7528)